python/paddle/fluid/tests/unittests/test_layers.py

```diff
@@ -173,6 +173,16 @@ def test_sigmoid_cross_entropy(self):
                     x=dat, label=lbl))
         print(str(program))
 
+    def test_hsigmoid(self):
+        program = Program()
+        with program_guard(program):
+            x = layers.data(name='x', shape=[2, 2], dtype='float32')
+            y = layers.data(name='y', shape=[1, 2], dtype='int64')
```

Comment:
Please refer to the data shape in `test_softmax_with_cross_entropy`. If defined as above, the actual shape of x will be `[-1, 1, 2]`.

Reply:
Done